codecademy/functions.py

```diff
@@ -47,3 +47,63 @@ def distance_from_zero_type(arg):
     else:
         return "Nope"
 # END 4. Function ex-19
+
+# 4. Function ex-1
+def tax(bill):
+    """Adds 8% tax to a restaurant bill."""
+    bill *= 1.08
+    print("With tax: %f" % bill)
+    return bill
+
+
+def tip(bill):
+    """Adds 15% tip to a restaurant bill."""
+    bill *= 1.15
+    print("With tip: %f" % bill)
+    return bill
+
+
+meal_cost = 100
+meal_with_tax = tax(meal_cost)
+meal_with_tip = tip(meal_with_tax)
```

Comment:
this file should only contain functions  - please comment out or remove all the code that makes calls to those functions - we will do this in the tests.
So this file is just like a library with functions.